test/e2e/e2e_toomanyrestarts_test.go

```diff
@@ -101,7 +101,6 @@ func TestTooManyRestarts(t *testing.T) {
 	if _, err := clientSet.CoreV1().Namespaces().Create(ctx, testNamespace, metav1.CreateOptions{}); err != nil {
 		t.Fatalf("Unable to create ns %v", testNamespace.Name)
 	}
-	defer clientSet.CoreV1().Namespaces().Delete(ctx, testNamespace.Name, metav1.DeleteOptions{})
```

Comment:
This needs to stay here in case creating a deployment errors and t.Fatalf is invoked.

Reply:
I didn't remove this logic. I move it to https://github.com/kubernetes-sigs/descheduler/blob/1e23dbcc2669f69288ec6c9dbcd51fa7b3c1c8e5/test/e2e/e2e_toomanyrestarts_test.go#L121